emapper.py

```diff
@@ -917,6 +1077,12 @@ def parse_args(parser):
     pg_diamond.add_argument('--gapextend', dest='gapextend', type=int, default=None, 
                     help='Gap extend  penalty')
 
+    pg_diamond.add_argument('--query-cover', dest='query_cover', type=int, default=60, 
+                    help='Report only alignments above the given percentage of query cover. Default=60')
```

Comment:
60 is 60%? make sure the parameter in diamond does not expect 0.6 instead

Reply:
also, are these params affecting diamond when used to functional annotation or only to predict orthologs?